docs/content/preview/secure/authentication/ldap-authentication-ysql.md

```diff
@@ -62,17 +62,17 @@ The configurations supported for simple bind mode.
 
 In `Search + Bind` mode, YB-Tserver will bind to the LDAP directory with a fixed username and password, specified with `ldapbinddn` and `ldapbindpasswd`, and performs a search for the user trying to log into the database. This mode is commonly used by LDAP authentication schemes in other software.
 
-For Searching the LDAP directory if no fixed username and password is configured at YB-TServer, an anonymous bind will be attempted to the directory. The search will be performed over the subtree at `ldapbasedn`, and will try to do an exact match of the attribute specified in `ldapsearchattribute`. After the user has been found in this search, the server disconnects and re-binds to the directory as this user, using the password specified by the client, to verify that the login is correct.
+For Searching the LDAP directory, if no fixed username and password is configured at YB-TServer, an anonymous bind will be attempted to the directory. The search is performed over the subtree at `ldapbasedn`, and tries to do an exact match of the attribute specified in `ldapsearchattribute`. After the user has been found in this search, the server disconnects and re-binds to the directory as this user, using the password specified by the client, to verify that the login is correct.
```

Comment:
- Why is S capitalize in Searching?
- username  >  user name
- at YB-TServer ?
- re-binds  >  rebinds
- After the user has been found in this search, the server disconnects and re-binds to the directory as this user, using the password specified by the client, to verify that the login is correct.  >  When the user is found, the server disconnects and rebinds to the directory as this user via the password specified by the client to verify that the login is correct.

The whole para sounds like implementation details.